### ISSUE TITLE
Creating a flow in the web-ui ignores json

### ISSUE THREAD

**Issue by cimabe** [openintegrationhub/openintegrationhub#1455]
**Describe the bug**
When creating a new flow in the web-ui (`/flows`) it always creates this default flow no matter the input.

<details>
    <summary>created flow</summary>

  ```json5
  {
    name: 'Add name here',
    description: 'Add description here',
    graph: {
      nodes: [
        {
          id: 'step_1',
          componentId: '...',
          function: 'TRIGGER',
          name: 'Flow node name',
          description: 'Flow node description'
        }
      ],
      edges: []
    },
    type: 'ordinary',
    owners: [
      {
        id: '...',
        type: 'user'
      }
    ],
    status: 'inactive',
    cron: '*/2 * * * *',
    createdAt: '...',
    updatedAt: '...',
    id: '...'
  }
  ```
</details>

**Expected behavior**
A new flow based on the json in the input should get created.

**To Reproduce**

1. Go to `/flow` in the web-ui
2. Press `ADD+`
3. write anything in the input e.g. `{}`
4. press the pen for the new flow
5. look at the flow json

**Your OIH Setup Information**
local via Minikube on Windows
openintegrationhub/web-ui `0.4.4`

**Comment by RobinBrinkmann** [comment]
Thanks for letting us know @cimabe - I could not reproduce this behaviour on my end yet, but we are looking into it.

**Comment by weberjm** [comment]
I have been able to reproduce in our test environment. It looks like this commit from February changed the `createFlow()` functionality to always use the `intiatialFlow` (initial Flow) data to save the flow, rather than the `editorData` object if the state was changed. 

https://github.com/openintegrationhub/openintegrationhub/commit/17abfbb5bbd1d185c9afc4a0db859c21fc0f1f75

Note:  the `intiatialFlow` object also does not match the placeholder data shown to the user, which is additional poor UX.
